spec/virtual_includes_spec.rb

```diff
@@ -386,11 +386,40 @@
     end
   end
 
+  describe ".eager_load" do
+    it "preloads standard associations (:books)" do
+      expect(Author.eager_load(:books)).to preload_values(:first_book_name, book_name)
+      expect(Author.eager_load([:books])).to preload_values(:first_book_name, book_name)
+      expect(Author.eager_load([[:books]])).to preload_values(:first_book_name, book_name)
+      expect(Author.eager_load(:books => {})).to preload_values(:first_book_name, book_name)
+    end
+
+    it "preloads associations (:uses => :books)" do
+      expect(Author.eager_load(:first_book_name)).to preload_values(:first_book_name, book_name)
+      expect(Author.eager_load([:first_book_name])).to preload_values(:first_book_name, book_name)
+      expect(Author.eager_load([[:first_book_name]])).to preload_values(:first_book_name, book_name)
+      expect(Author.eager_load(:first_book_name => {})).to preload_values(:first_book_name, book_name)
+    end
+
+    it "preloads standard associations (:books)" do
+      expect(Author.eager_load(:books)).to preload_values(:first_book_name, book_name)
+      expect(Author.eager_load([:books])).to preload_values(:first_book_name, book_name)
+      expect(Author.eager_load([[:books]])).to preload_values(:first_book_name, book_name)
+      expect(Author.eager_load(:books => {})).to preload_values(:first_book_name, book_name)
+    end
```

Comment:
This test looks like a straight duplicate of the one at the beginning of this describe.

Reply:
yea, testing eager loading, which should work the same as `includes`.
To be honest, we should be using `eagerload` and drop our `includes().references()`, but we'll get there when we get there.